mrtt-ui/src/hooks/useLocalStorage.js

```diff
@@ -0,0 +1,24 @@
+import { useState } from 'react'
+
+export const useLocalStorage = (keyName, defaultValue) => {
+  const [storedValue, setStoredValue] = useState(() => {
```

Comment:
Im not sure we need to make this a hook rather than a utility function. Im not sure we need to use react state to store the values either, we can use localstorage for that no?

Its better to keep code react-free if possible to make it more maintainable and simple.

That said, I think this as-is is good enough. Just a comment.

Reply:
This is initially implemented as a library but "destroying" the token when logging out doesn't update the components as expected. Feel free to suggest a re-implementation if you feel this is an overkill.